xmake/rules/c++/modules/modules_support/clang/compiler_support.lua

```diff
@@ -201,7 +203,18 @@ function get_stdmodules(target)
         local cpplib = _get_cpplibrary_name(target)
         if cpplib then
             if cpplib == "c++" then
-                -- TODO support libc++ std module file when https://github.com/xmake-io/xmake/pull/4630
+                local clang_path = path.directory(get_clang_path(target))
+                local clang_lib_path = path.normalize(path.join(clang_path, "..", "lib"))
+                local modules_json_path = os.files(path.join(clang_lib_path, "**.modules.json"))[1]
```

Comment:
we should use find_file, `os.files` maybe will return table. and find_file will be faster.

Reply:
and please add some comments and relative links about clang libc++ module